README.md

```diff
@@ -3,25 +3,28 @@ This bundle provides a Mautic Housekeeping Command for database cleanup purposes
 
 ## Command
 
-Command to delete lead_event_log, campaign_lead_event_log, email_stats and email_stats_devices table entries. 
+Command to delete lead_event_log table entries, campaign_lead_event_log table entries, email_stats table entries that referenced emails entry is currently not published and email_stats_devices table entries.
```

Comment:
Correction: "Command to delete lead_event_log, campaign_lead_event_log, email_stats_devices and email_stats table entries where the referenced email entry is currently not published."

Reply:
Thanks @Moongazer for this spelling recommendation - implemented that with my current commit.